src/main/scala/org/alvearie/hri/api/MgmtClient.scala

```diff
@@ -159,4 +162,44 @@ object MgmtClient {
   val hriConsumerScope = "hri_consumer"
   val accessTokenField = "access_token"
   val audienceField = "audience"
+
+  val trustStoreEnv = "HRI_TRUSTSTORE"
+  val trustStorePasswordEnv = "HRI_TRUSTSTORE_PASSWORD"
+
+  /**
+   * If 'HRI_TRUSTSTORE' and 'HRI_TRUSTSTORE_PASSWORD' are set, constructs an Http client using the specified trust
+   * store. If not set, creates a default Http client.
+   * If unable to load the trust store or create the client, an Exception is thrown.
+   * @return a Http client
+   */
+  def createHttpClient() : CloseableHttpClient = {
+    val log = LoggerFactory.getLogger(this.getClass)
+    val trustStorePath = System.getenv(trustStoreEnv)
+    val password = System.getenv(trustStorePasswordEnv)
+
+    if( trustStorePath == null || trustStorePath.isEmpty ) {
+      log.info("HRI_TRUSTSTORE is not set, so creating default Http client")
+      return HttpClients.createDefault()
+    } else if ( password == null || password.isEmpty ) {
+      val msg = trustStoreEnv + " is set, but " + trustStorePasswordEnv + " is not. Both must be empty or set."
+      log.error(msg)
+      throw new IllegalArgumentException(msg)
+    }
+    log.info("Creating Http client with trust store {}", trustStorePath)
+
+    val trustStoreFile = new File(trustStorePath);
+    if (!trustStoreFile.exists() || !trustStoreFile.isFile) {
+      val msg = "Not found or not a file: " + trustStoreFile.getPath
+      log.error(msg);
+      throw new FileNotFoundException(msg);
+    }
```

Comment:
This new method should not be included in the `support-2.x` branch of the code.

You are going to want to compare this alvearie versin of the file with the private Repo version --> https://github.ibm.com/wffh-hri/flink-pipeline-core/blob/support-2.x/src/main/scala/com/ibm/watson/health/foundation/hri/api/MgmtClient.scala

Reply:
I've removed the method and compared with `2.x`, it should be matching now